e2e/tests/session.feature

```diff
@@ -7,8 +7,7 @@ Feature: Session Namespace
     Then the claim created by supplier "supplier1" for service "svc1" for application "app1" should be persisted on-chain
     # TODO_BLOCKER(@bryanchriswhite): And a cosmos-sdk event (e.g. EventClaimCreated) should be emitted.
     And the user should wait for the "proof" module "SubmitProof" Message to be submitted
-    # TODO_BLOCKER(@bryanchriswhite): And a cosmos-sdk event (e.g. EventClaimCreated) should be emitted.
```

Comment:
Did you intend to kill this comment? 

I think we still want to assert that an event is emitted as a result of proof submission. This will cover regression part of #580.

Reply:
@bryanchriswhite, isn't the proof submission part covered by the step above?
`And the user should wait for the "proof" module "SubmitProof" Message to be submitted`

Reply:
@bryanchriswhite Yes.

1. It is a duplicate of line 8
2. The new step I added is event driven